src/main/java/uk/gov/companieshouse/efs/api/events/service/DelayedSubmissionHandlerContext.java

```diff
@@ -6,9 +6,13 @@
 import java.util.stream.Collectors;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
 @Component
 public class DelayedSubmissionHandlerContext {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger("efs-submission-api");
```

Comment:
Rather than creating a constant here, I'd recommend injecting the existing Logger bean in the constructor

Reply:
Ok can do, but we use a LOOGER constant in lots of other places in this project.  Each time setting the namespace.

Reply:
done as recommended.